packages/simple-types/src/simple-types.ts

```diff
@@ -149,8 +149,8 @@ export function convertItemToTemplate(
               break;
             case "form":
               itemTemplate.dependencies = itemTemplate.dependencies.concat(
-                relatedItemsResponse.relatedItems.map(
-                  relatedItem => relatedItem.id
+                (relatedItemsResponse as any).relatedItems.map(
```

Comment:
This line confuses me. If it's any how do we know it has a relatedItems array?

Reply:
@MikeTschudi since we are now at TypeScript 3.7 can we use the new optional chaining for stuff like this? I have not played with it yet just saw someone mention in the devs slack.

`(relatedItemsResponse as any).relatedItems?.map(`

Reply:
I see the catch that will ensure it's there on error...does the portal query also garuntee it? Just curious.

Reply:
Thanks for pointing out optional chaining--that'll be useful. In this change, the statement compiles in VS Code, but fails during `npm build`, so I added the cast to `any`.

Reply:
What are you referring to for the catch & portal query, please?

Reply:
@MikeTschudi I was just talking about the fact that I could see that the catch handling would guarantee that it had a relatedItems array but couldn’t see that it necessarily would with the underlying portal query.

`Promise.all([
    dataPromise,
    resourcePromise,
    relatedPromise.catch(
        () =>
        ({ total: 0, relatedItems: [] } as common.IGetRelatedItemsResponse)
    )
]).then(`